pkgs/development/ocaml-modules/git/mirage.nix

```diff
@@ -0,0 +1,92 @@
+{ lib, buildDunePackage
+, git
+, mimic
+, mimic-happy-eyeballs
+, base64
+, git-paf
+, awa
+, awa-mirage
+, dns
+, dns-client
+, tls
+, tls-mirage
+, uri
+, hex
+, happy-eyeballs-mirage
+, happy-eyeballs
+, ca-certs-nss
+, mirage-crypto
+, ptime
+, x509
+, cstruct
+, tcpip
+, domain-name
+, fmt
+, ipaddr
+, lwt
+, mirage-clock
+, mirage-flow
+, mirage-random
+, mirage-time
+, result
+, rresult
+, alcotest
+, alcotest-lwt
+, bigstringaf
+, logs
+, ke
+}:
+
+buildDunePackage {
+  pname = "git-mirage";
+
+  inherit (git) version src;
+
+  minimalOCamlVersion = "4.08";
+
+  propagatedBuildInputs = [
+    git
+    mimic
+    mimic-happy-eyeballs
+    base64
+    git-paf
+    awa
+    awa-mirage
+    dns
+    dns-client
+    tls
+    tls-mirage
+    uri
+    hex
+    happy-eyeballs-mirage
+    happy-eyeballs
+    ca-certs-nss
+    mirage-crypto
+    ptime
+    x509
+    cstruct
+    tcpip
+    domain-name
+    fmt
+    ipaddr
+    lwt
+    mirage-clock
+    mirage-flow
+    mirage-random
+    mirage-time
+    result
+    rresult
+  ];
```

Comment:
At `git/mirage.nix`:

```
  buildInputs = [
    dns
    dns-client
    happy-eyeballs-mirage
    ipaddr
    mirage-random
    rresult
  ];

  propagatedBuildInputs = [
    git
    mimic
    mimic-happy-eyeballs
    base64
    git-paf
    awa
    awa-mirage
    tls
    tls-mirage
    uri
    hex
    happy-eyeballs
    ca-certs-nss
    mirage-crypto
    ptime
    x509
    cstruct
    tcpip
    domain-name
    fmt
    lwt
    mirage-clock
    mirage-flow
    mirage-time
    result
  ];
```

The dependencies:
* dns
* dns-client
* happy-eyeballs-mirage
* ipaddr
* mirage-random
* rresult

Are not listed at: `result/lib/ocaml/4.14.0/site-lib/git-mirage/META`
I guess they should be at `buildInputs` instead